tests/gis_tests/relatedapp/tests.py

```diff
@@ -346,9 +346,12 @@ def test_centroid_collect_filter(self):
             )
         )
         city = qs.get(name="Aurora")
-        self.assertIsInstance(city.parcel_centroid, Point)
-        self.assertAlmostEqual(city.parcel_centroid[0], 3.2128, 4)
-        self.assertAlmostEqual(city.parcel_centroid[1], 1.5, 4)
+        if connection.ops.mariadb:
+            self.assertIsNone(city.parcel_centroid)
```

Comment:
Do you think there might be an issue with the Collect aggregation support?
Just conscious that 12.0.1 is not yet their stable release and not sure if they will pull the feature again if they find any issues

Reverting isn't very difficult though and I suppose we might want to merge before the 6.0 alpha (12.0.2 should be in ~3 months so waiting might miss it). Perhaps it's just something to keep an eye on?

Reply:
I passed the same value directly to  `Centroid` and the result is the same so it has nothing to do with `Collect()`:
```python
>>> qs = SomeModel.objects.annotate(
...     multipoint=Value(MultiPoint(Point(1.7128, -2.006), Point(4.7128, 5.006), srid=4326), output_field=MultiPointField()),
...     centroid=Centroid(F("multipoint"))
... )
>>> obj = qs.first()
>>> obj.centroid
None
>>> obj.multipoint
SRID=4326;MULTIPOINT (1.7128 -2.006, 4.7128 5.006)
```